app/components/ui/molecules/DashboardList.md

```diff
@@ -0,0 +1,23 @@
+Dashboard list of submissions
+
+```js
+;<DashboardList
+  manuscripts={[
```

Comment:
Could you also add the other view of this component to this markdown file - the `if (!manuscripts.length)` version?